docs/draft/notifications.md

```diff
@@ -0,0 +1,49 @@
+---
+layout: default
+title: Notifications
+nav_order: 2
+has_children: true
+permalink: /specs/draft
+---
+
+# Notifications in Status
+
+## Why is there no Push Notifications?
+
+Push Notifications, as offered by Apple and Google are a privacy concern, they require a centralized service that is aware of who the notification needs to be delivered to.
+
+An early implementation of push notifications was done using the Firebase library which required users to exchange notifications ids and send notifications through Firebase. This was fairly limited by the fact that one could only send/receive a notification after exchanging these ids with the contact. We were also not satisfied with the fact that it was a huge dependency, which required the installation of `Play Services` that some users weren't comfortable with. 
+
+## Local Notifications
+
+Using local notifications, we can offer notifications for any event in the app without any privacy cost and dependency on third party services.
+
+### Android
+
+Android allows running services on the device. When the user enables notifications, the client starts a `Foreground Service`, and a permanent notification indicating that the service is running is displayed to the user, as required by Android guidelines.
```

Comment:
This currently reads more like "the (one) client does this". Ideally it should read something like: "If a client wants to support notifications, it MUST do X, Y, Z etc.

Then less emphasis on specific implementation details, other than what is relevant for all implementations (e.g. Nim based, greenfield, other Go impl, desktop one, etc etc)